src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -288,7 +310,9 @@ internal override string GetToolName()
         /// <param name="consoleOutput"></param>
         public override void PrintHelp(TextWriter consoleOutput)
         {
-            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture));
+            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture),
+                LanguageVersion.Default.MapSpecifiedToEffectiveVersion().ToDisplayString(),
+                LanguageVersion.Latest.MapSpecifiedToEffectiveVersion().ToDisplayString());
```

Comment:
At the moment, I agree. But once we reach C# 8 or 9, it would be funny to list 7 and 7.1.